src/main/java/sirius/biz/storage/layer2/mongo/MongoBlob.java

```diff
@@ -524,4 +530,14 @@ public LocalDateTime getLastTouched() {
     public void setLastTouched(LocalDateTime lastTouched) {
         this.lastTouched = lastTouched;
     }
+
+    @Override
+    public boolean isReadOnly() {
+        return readOnly;
+    }
+
+    @Override
+    public void setReadOnly(boolean readOnly) {
+        getStorageSpace().setReadOnly(this, readOnly);
```

Comment:
This sounds like a simple setter but it actually triggers an update query via the storage space. Is this a good combination?
Other setters of this class are just that - simple setters.

Reply:
A few other actions are processed like this: update, rename, touch...
Either another method name (which sounds less like a setter), or 2 methods (lock / unlock)?